packages/legacy/src/Providers/components/AddEditProviderModal/AddEditProviderModal.tsx

```diff
@@ -139,9 +147,21 @@ const useAddProviderFormState = (
       caCert: useFormField('', yup.string().label('CA certificate').required()),
       caCertFilename: useFormField('', yup.string()),
     }),
+    openstack: useFormState({
+      ...commonProviderFields,
+      openstackUrl: useFormField('', urlSchema.required().label(getLabelName('URL', brandPrefix(providerTypeField.value)))),
```

Comment:
```suggestion
      openstackUrl: useFormField('', urlSchema.required().label('Identity service (Keystone) server URL')),
```

Reply:
This `brandPrefix` is for setting all prefixes per provider in one place for all fields (url, username and password) https://github.com/kubev2v/forklift-console-plugin/pull/206/files#diff-5fa98e7b93430fe42c701b496a43bff5d4461dab4a7a74813c127e36894839e3R60 
so I prefer leaving it like that